doc/guides/dts/index.rst

```diff
@@ -457,6 +457,50 @@ hardware resources:
       uart-1 = &uart1;
    };
 
+To have a generic name in various Zephyr sample and test code we define some
+common aliases.
+
+The full set of Zephyr-specific ``aliases`` properties follows:
+
+.. list-table::
+   :header-rows: 1
+
+   * - ``aliases`` property name
+     - Generated symbol
```

Comment:
From this, I expected the second column to list the `DT_*` macro name. Was that the original plan?

Should probably change the heading to something like "Description" otherwise.